src/Lean/Parser/Term.lean

```diff
@@ -755,6 +755,42 @@ the syntax `partial_fixpoint monotonicity by $tac` the proof can be done manuall
     optional (checkColGt "indentation" >> nonReservedSymbol "monotonicity " >>
               checkColGt "indented monotonicity proof" >> termParser))
 
+/--
+Defines a coinductive predicate using lattice theory, based on the Knaster-Tarski fixpoint theorem.
+
+This feature constructs coinductive predicates by leveraging the lattice structure on `Prop`
+(as provided by `Lean.Order.complete_lattice`) and ensures correctness through monotonicity
+(as defined in `Lean.Order.monotone`).
```

Comment:
The `Lean.Order` definitions are somewhat internal, so we maybe shouldn’t mention them in this user-facing docstring.